### ISSUE TITLE
external access to a port?

### ISSUE THREAD

**Issue by ezar** [jotyGill/openpyn-nordvpn#140]
If I want to access to port from external network, how I configure it?
I have a nginx and I want to access!
Regards,

**Comment by yuriw** [comment]
@ezar  You will need to do "port forwarding" on your router.

**Comment by ezar** [comment]
Yes! I have a port forwarding .. but when I connect to nord, I can’t connect to this port!

**Comment by ezar** [comment]
I make it ;)

**Comment by Njur** [comment]
Will it be possible to somehow open ports? I really want port forwarding.

**Comment by blindguy** [comment]
I would love to be able to use this feature to connect to my box form the outside for various ports.

I use the -F feature to connect from my internal network but I can't access anything from outside.
Maybe a -Z for external ports?

I have port forwarding on my router and everythgin works if i disconnect from the vpn.
